README.md

```diff
@@ -58,12 +63,15 @@ The tools have dependencies on other tools, and there are also typically 2 ways
  - one that is not pre-reconfigured and therefore does require autoconf/automake and associated tools
 
 To build from scratch, start with the tarballs of the following tools:
- - m4: requires m4, curl in boot and xlclang installed on the system.
- - perl: additionally requires make, git in boot and m4 in prod.
- - make: additionally requires perl in prod for running test cases.
- - libz: additional requires make in prod
```

Comment:
@MikeFultonDev ... I presume this should be _zlib_ ?

Reply:
Yes.

Reply:
corrected